projects/packages/search/src/instant-search/components/search-filters.scss

```diff
@@ -19,8 +19,6 @@
 }
 
 .jetpack-instant-search__clear-filters-link {
-	@include inherit-parent-colors-important();
-	@include inherit-parent-font-size-important();
 	border: none;
```

Comment:
Is there any particular reason we didn't put these in the same selector on L36?

Reply:
I'm not sure about that. Those changes were made while I was away so I wasn't part of any discussions that might have happened there. It was a bit of a rush to get the visual fixes into an Atomic update, I think.

That said, I could see an argument for keeping font metrics and colours as separate mixins.